features/preferences/impl/src/main/kotlin/io/element/android/features/preferences/impl/tasks/ClearCacheUseCase.kt

```diff
@@ -43,6 +44,7 @@ class DefaultClearCacheUseCase @Inject constructor(
     private val coroutineDispatchers: CoroutineDispatchers,
     private val defaultCacheIndexProvider: DefaultCacheService,
     private val okHttpClient: Provider<OkHttpClient>,
+    private val ftueState: Provider<FtueState>,
```

Comment:
It's compiling, but crashing in the room list if I use the `FtueState` type directly, not sure why :/

Reply:
It's working fine for me without the `Provider`. Maybe I'm doing something wrong? What did you do reproduce the issue?

Reply:
I will try again, but I had the classical `Cannot find NodeFactory for RoomFlowNode` issue.

Reply:
It's working today, I have dropped the commit which introduced this.